packages/blitz-rpc/src/index-server.ts

```diff
@@ -192,11 +212,22 @@ export function rpcHandler(config: RpcConfig) {
       }
 
       try {
-        const data = deserialize({
-          json: req.body.params,
-          meta: req.body.meta?.params,
-        })
-
+        let data
+        if (req.method === "POST") {
+          data = deserialize({
+            json: req.body.params,
+            meta: req.body.meta?.params,
+          })
+        } else {
+          let meta = req.query.meta
+          if (Array.isArray(meta)) {
+            meta = meta[meta.length - 1]
+          }
```

Comment:
can you add a comment describing why this is necessary?

Reply:
@flybayer this was due to an error when I tested this feature in a slightly bigger project with multiple queries set as `GET`
The error was 
![image](https://user-images.githubusercontent.com/83594610/195668698-18291797-e6e9-49b5-8842-2d2146ce00d6.png)
Then during debugging I found that it is because of the `?params` words added to the `meta` properties that are not being parsed by `superjson`
![image](https://user-images.githubusercontent.com/83594610/195668988-0f861776-a1bc-49ec-bebe-d7c6efa8ae39.png)
This was my workaround to use only the first meta value added to avoid this error. 
If this can be fixed in a better way, I shall add those changes.

Reply:
ok that's not good, then we should fix `meta` ever being an array.